Source/Core/VideoBackends/Software/SWVertexLoader.h

```diff
@@ -37,7 +37,7 @@ class SWVertexLoader : public VertexManagerBase
 
   void ParseVertex(const PortableVertexDeclaration& vdec, int index);
 
-  SetupUnit* m_SetupUnit;
+  std::unique_ptr<SetupUnit> m_SetupUnit;
```

Comment:
Alright, I just opted for just constructing it as a direct object instead of using pointers in that case.